benchmarks/span-labeling-datasets/scripts/preprocess.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+from string import digits
+
+import typer
+from wasabi import msg
+
+
+def preprocess(
+    input_path: Path, output_path: Path, *, strip_digits: bool = False
+) -> None:
+    """
+    Helper function to canonicalize all datasets into the same ConLL format.
+    """
+    with input_path.open(encoding="utf-8") as f:
```

Comment:
Added encoding here so that `UnicodeDecodeError` doesn't occur for Windows. Same goes for file output